docs/docs/platform/preferences.md

```diff
@@ -2,31 +2,208 @@
+4. Template level channel preferences will be applied to all subscribers by default unless subscriber overrides them.
+
+`Example`
+
+1. For `First Template`, `critical` is false, `enabled` is true, in_app, email, push channels are true and sms, chat channels are false. As per above order of priority of preferences, notification will be filtered as per subscriber preferences. Subscriber will receive in-app, email and push notifications, but not receive sms and chat notifications.
+
+2. For `Second Template`, `critical` is false, `enabled` is false, all channels are true. As per above order of priority of preferences, subscriber will not receive any type of notification because all channels becomne disable due to false value of `enabled` field.
+
+3. For `Third Template`. `critical` is true, `enabled` is false, email, chat channels are false and rest three channels are true. As per above order of priority of preferences, subscriber will receive all type of notifications as this template is `critical`.
+
+```json title="Subscriber preference example for three templates"
+[
+  {
+    "template": {
+      "_id": "firstTemplateIdentifier",
+      "name": "First Template",
+      "critical": false
+    },
+    "preference": {
+      "enabled": true,
+      "channels": {
+        "in_app": true,
+        "email": true,
+        "sms": false,
+        "push": true,
+        "chat": false
+      }
+    }
+  },
+  {
+    "template": {
+      "_id": "secondTemplateIdentifier",
+      "name": "Second Template",
+      "critical": false
+    },
+    "preference": {
+      "enabled": false,
+      "channels": {
+        "in_app": true,
+        "email": true,
+        "sms": true,
+        "push": true,
+        "chat": true
+      }
+    }
+  },
+  {
+    "template": {
+      "_id": "thirdTemplateIdentifier",
+      "name": "Third Template",
+      "critical": true
+    },
+    "preference": {
+      "enabled": false,
+      "channels": {
+        "in_app": true,
+        "email": false,
+        "sms": true,
+        "push": true,
+        "chat": false
+      }
+    }
+  }
+]
+```
+
+## Important Links
+
+- [Get subscriber preferences API](https://docs.novu.co/api/get-subscriber-preferences/)
+- [Update subscriber preference for a template API](https://docs.novu.co/api/update-subscriber-preference/)
+- [Update template API](https://docs.novu.co/api/update-notification-template/)
+
+## Frequently Asked Questions
+
+<details>
+  <summary>How to change <code>enable</code> field from UI?</summary>
+  <p>This field can only be changed using API.</p>
+</details>
+
+<details>
+  <summary>What preferences are applied to subscriber when we create a new template?</summary>
+  <p>Subscriber will inherit all preferences from template in case of new template. After subsequent preferences update, subscriber preferences will not inherit template-level preferences.</p>
```

Comment:
=> In the case of a new template, the subscriber will inherit all preferences from the template. However, after subsequent preference updates, the subscriber's preferences will not inherit template-level preferences.